todo_app/data/trello_items.py

```diff
@@ -2,10 +2,8 @@
 from todo_app.data.helpers import map_list_id_to_status
 from todo_app.item import Item
 from todo_app.trello_api import TrelloApi
-
-trello_api = TrelloApi(os.getenv('TRELLO_API_BASE_URL'), os.getenv('TRELLO_BOARD_ID'), os.getenv('TRELLO_API_TOKEN'), os.getenv('TRELLO_API_KEY'))
-
 def get_items():
+    trello_api = TrelloApi(os.getenv('TRELLO_API_BASE_URL'), os.getenv('TRELLO_BOARD_ID'), os.getenv('TRELLO_API_TOKEN'), os.getenv('TRELLO_API_KEY'))
```

Comment:
This was my workaround for ensuring test credentials are picked up, although from a performance perspective I'm not sure it's ideal to re-instantiate the API class each method call?

Reply:
Looking at what the class does on instantiation, I don't think it's a significant cost at all really, but if we wanted to reduce that, we could similarly turn this file into a class so that we can share that:

```python
class ItemService():
    def __init__(self):
        self.trello_api = TrelloApi(os.getenv('TRELLO_API_BASE_URL'), os.getenv('TRELLO_BOARD_ID'), os.getenv('TRELLO_API_TOKEN'), os.getenv('TRELLO_API_KEY'))

    def get_items(self):
        board_lists = self.trello_api.get_cards()
        items = []
        ...
```

We can then instantiate a single ItemService in `app.py` within the `create_app` function